SharedPackages/BrowserServicesKit/Tests/PixelExperimentKitTests/PixelExperimentKitTests.swift

```diff
@@ -587,56 +588,3 @@ final class PixelExperimentKitTests: XCTestCase {
     }
 
 }
-
-class MockExperimentActionPixelStore: ExperimentActionPixelStore {
```

Comment:
Please refer to this comment for more information on this change: https://github.com/duckduckgo/apple-browsers/pull/562/files#r2055883698

I needed to make these mocks available to the main app.